Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -12,11 +12,13 @@ const { vtkWarningMacro, vtkErrorMacro, normalizeWheel, vtkOnceErrorMacro } =
 // ----------------------------------------------------------------------------
 
 const deviceInputMap = {
-  'OpenVR Gamepad': [
-    Input.TrackPad,
+  'xr-standard': [
```

Comment:
Where does this get used?

Reply:
In [line 451](https://github.com/Kitware/vtk-js/pull/2184/files#diff-6f8b2d910dfd95c1cc6d62e23311ede827ceae8efc6ad2b705fff3c15a01f3b3R451):
```js
input:
                deviceInputMap[gp.mapping] && deviceInputMap[gp.mapping][b]
                  ? deviceInputMap[gp.mapping][b]
                  : Input.Trigger,
```

Essentially we need to query what gamepad standard we are using so that we can translate from an ordered list of button inputs into meaningful enums about which buttons were pressed. I believe the idea of `deviceInputMap` is that we could extend this for other standards in the future. The [Gamepad API](https://www.w3.org/TR/gamepad/#dom-gamepadmappingtype) outlines the difference between the "standard" and "xr-standard" gamepad layouts.

Reply:
@floryst Did this resolve your question?

Reply:
Yep, thanks!